atomic_reactor/plugins/rpmqa.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, workflow, ignore_autogenerated_gpg_keys=True):
         :param workflow: DockerBuildWorkflow instance
         """
         # call parent constructor
-        super(PostBuildRPMqaPlugin, self).__init__(workflow)
+        super().__init__(workflow)
```

Comment:
since we're doing this I don't think we need the `__all__`  anymore.

Reply:
Why? I don't think the usage of `__all__` is relative to this change. `__all__` affects what are exported when use `from ... import *`.